Spectrum_Peak_Visualization.py

```diff
@@ -0,0 +1,108 @@
+import matplotlib.pyplot as plt
+import SPEFile
+from ROI_Maker import ROI_Maker
+
+def plot_spectrum(spectrum, title= None, energy_range= None):
+    """
+    plot_spectrum plots the spectra only.
+    """
+    zero_offset = spectrum.energy_cal[0]
+    energy_per_channel = spectrum.energy_cal[1]
+    energy_axis = zero_offset + energy_per_channel*spectrum.channel
+    counts = spectrum.data
+
+    plt.semilogy(energy_axis, counts, drawstyle='steps-mid')
+    plt.xlabel('Energy (keV)')
+    plt.ylabel('Counts')
+    
+    if title != None:      
```

Comment:
When checking for a `None` value, use the `is` operator. This is [in PEP 8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations), reads nicer, and in some cases it is possible that `!=` could not work as expected.

So:
`if title is not None:`